src/test/resources/mapping/expectedTransformationFields.json

```
@@ -1,698 +1,875 @@
+      "path": "$.holdings.electronicAccess[?(@.relationshipId==3b430592-2e09-4b48-9a0c-0636d66b9fb3)].uri",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.hrid",
+      "displayNameKey": "holdings.hrid",
+      "path": "$.holdings.hrid",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.id",
+      "displayNameKey": "holdings.id",
+      "path": "$.holdings.id",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.instanceid",
+      "displayNameKey": "holdings.instanceId",
+      "path": "$.holdings.instanceId",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.metadata.createdbyuserid",
+      "displayNameKey": "holdings.metadata.createdByUserId",
+      "path": "$.holdings.metadata.createdByUserId",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.metadata.createddate",
+      "displayNameKey": "holdings.metadata.createdDate",
+      "path": "$.holdings.metadata.createdDate",
+      "recordType": "HOLDINGS",
+      "metadataParameters": {
+        "datesOfPublication": "$.instance.publication[*].dateOfPublication",
+        "languages": "$.instance.languages"
+      }
+    },
+    {
+      "fieldId": "holdings.metadata.updatedbyuserid",
+      "displayNameKey": "holdings.metadata.updatedByUserId",
+      "path": "$.holdings.metadata.updatedByUserId",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.metadata.updateddate",
+      "displayNameKey": "holdings.metadata.updatedDate",
+      "path": "$.holdings.metadata.updatedDate",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.holdingnotetypeid.copy.note",
+      "displayNameKey": "holdings.holdingNoteTypeId",
+      "referenceDataValue": "Copy note",
+      "path": "$.holdings.notes[?(@.holdingsNoteTypeId==c4407cc7-d79f-4609-95bd-1cefb2e2b5c5) && ?(!(@.staffOnly) || @.staffOnly == false))]",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.holdingnotetypeid.staffonly.copy.note",
+      "displayNameKey": "holdings.holdingNoteTypeId.staffOnly",
+      "referenceDataValue": "Copy note - Staff only",
+      "path": "$.holdings.notes[?(@.holdingsNoteTypeId==c4407cc7-d79f-4609-95bd-1cefb2e2b5c5) && ?(!(@.staffOnly) || @.staffOnly == true))]",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.holdingnotetypeid.electronic.bookplate",
+      "displayNameKey": "holdings.holdingNoteTypeId",
+      "referenceDataValue": "Electronic bookplate",
+      "path": "$.holdings.notes[?(@.holdingsNoteTypeId==88914775-f677-4759-b57b-1a33b90b24e0) && ?(!(@.staffOnly) || @.staffOnly == false))]",
+      "recordType": "HOLDINGS"
+    },
+    {
+      "fieldId": "holdings.holdingnotetypeid.staffonly.electronic.bookplate",
+      "displayNameKey": "holdings.holdingNoteTypeId.staffOnly",
+      "referenceDataValue": "Electronic bookplate - Staff only",
```

Should it 
```suggestion
      "referenceDataValue": "Electronic bookplate",
```
`Statff only` is going to be added on UI.

Item - Notes - <itemNoteType .name> - Staff only

Please handle similar cases.